llvm/lib/Target/X86/X86SchedSapphireRapids.td

```diff
@@ -78,6 +80,10 @@ def SPRPort02_03_07_08_11 : ProcResGroup<[SPRPort02, SPRPort03, SPRPort07,
   let BufferSize = 72;
 }
 
+def SPRPortAny : ProcResGroup<[SPRPort00, SPRPort01, SPRPort02, SPRPort03,
```

Comment:
Why do we need SPRPortAny ?

Reply:
We get an assertion failure if we do not have a group that encompasses all the ports due to how they are layered currently.